salt/metalk8s/addons/ui/deployed.sls

```diff
@@ -39,7 +39,7 @@ Create metalk8s-ui service:
           protocol: TCP
           targetPort: 80
         selector:
-          k8s-app: ui
+          k8s-app: metalk8s-ui
```

Comment:
This should be `app`, not `k8s-app`, and the appropriate changes should be made to the Pod labels.